pod-operation/src/components/lim_temperature.rs

```diff
@@ -18,9 +19,8 @@ const BETA: f32 = 3950.0; // Kelvins
 const R_0: f32 = 10000.0; // Ohms
 const ROOM_TEMP: f32 = 25.0 + C_TO_K_CONVERSION; // Kelvins
 
-fn voltage_to_temp(voltage: i16) -> f32 {
-	let voltage = f32::from(voltage) / 1000.0;
-	let thermistor_resistance = (voltage * DIVIDER_RESISTANCE) / (V_IN - voltage);
+fn voltage_to_temp(voltage: f32) -> f32 {
+	let thermistor_resistance = ((V_IN - voltage) * DIVIDER_RESISTANCE) / voltage;
```

Comment:
Issue: with the inverted divider circuit, the ADC will clip as temperature increases rather than decreases. With the current resistor and FSR configuration, the maximum temperature we can read is about 42°C. We'll need to either increase the FSR again or change out the divider resistance value in the circuit.

Reply:
Chris confirmed we'll be switching to 1 kΩ resistors. We'll want to change the FSR back to standard (2.048 V) with 1 mV LSB and update the divider resistance constant.

Reply:
Note: still need to revert FSR and LSB.